app/models/concerns/sluggable.rb

```diff
@@ -9,6 +9,7 @@ module Sluggable
   end
 
   def slugify
+    return if slug.present?
     self.slug = title.parameterize
   end
```

Comment:
@tarebyte I added this line so that we can still create the assignment even if the `slug` is not present in the form values. What do we think of this?